### PR TITLE
Replace references to "Presto" and "SEP" in Delta Lake connector

### PR DESCRIPTION
**Pull request by jirassimok**
We missed a few references to "Presto" and "SEP" in the Delta Lake connector; I've fixed these to say "Trino."

### CODE REVIEW

File: plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeTableProperties.java [hunk 1]
```diff
@@ -43,7 +43,7 @@
     //       to pass the column names to ConnectorMetadata#getStatisticsCollectionMetadata; we should consider passing
     //       ConnectorTableHandle instead of ConnectorTableMetadata as an argument since it makes more information
     //       available (including the names of the columns to be analyzed)
-    public static final String ANALYZE_COLUMNS_PROPERTY = "$sep.analyze_columns";
+    public static final String ANALYZE_COLUMNS_PROPERTY = "$trino.analyze_columns";
```
Comment (hashhar) [inline plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeTableProperties.java:46]: Any impact?

**Review by hashhar — APPROVED** [review]
LGTM % question

File: plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplitSource.java [hunk 2]
```diff
@@ -73,7 +73,7 @@ public DeltaLakeSplitSource(
         this.dynamicFilter = requireNonNull(dynamicFilter, "dynamicFilter is null");
         queueSplits(splits, queue, executor)
                 .exceptionally(throwable -> {
-                    // set prestoException before finishing the queue to ensure failure is observed instead of successful completion
+                    // set trinoException before finishing the queue to ensure failure is observed instead of successful completion
```
Comment (findinpath) [inline plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplitSource.java:76]: nit: TrinoException

File: plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeTableProperties.java (hunk 1 above)
Comment (findinpath) [reply plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeTableProperties.java:46]: From looking at the code the this property is used only during the `ANALYZE` command. So I assume that it should have no impact.

@alexjo2144 please do also check this one to be make sure that I'm not mistaken.

**Review by findinpath — APPROVED** [review]

File: plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeTableProperties.java (hunk 1 above)
Comment (alexjo2144) [reply plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeTableProperties.java:46]: It should be fine, it's a transient property key that shouldn't be stored in HMS

File: plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplitSource.java (hunk 2 above)
Comment (jirassimok) [reply plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplitSource.java:76]: It's a variable's name.

**Review by hashhar — APPROVED** [review]

**Pull request merged**